src/taskgraph/main.py

```diff
@@ -838,7 +844,7 @@ def main(args=sys.argv[1:]):
     parser = create_parser()
     args = parser.parse_args(args)
     try:
-        args.command(vars(args))
+        return args.command(vars(args))
```

Comment:
should this be `sys.exit(args.command(...))` ?

Reply:
Possibly, either that or we should be returning 1 in the error case instead of raising `SystemExit`. This function is kind of built to be called programmatically, so I'm leaning towards the latter..

That said, switching to be consistent either way would technically be backwards incompatible, so I wasn't going to bother with it (at least in this PR).